### DISCUSSION TITLE
help

### DISCUSSION THREAD

**Discussion by cjmarklll** [anuraghazra/github-readme-stats#3859] [Q&A] [answered]
### Describe the bug

![image](https://github.com/user-attachments/assets/e8863813-2af1-4663-a81b-45aae7d487a9)
I deployed the project following the tutorial but an error occurred and I was sure the username was correct

**Comment by qwerty541** [comment, marked as answer]
Hey @cjmarklll. This error measn that something wrong with your GitHub API token. It can be expired or maybe you set your token to the environment variable with wrong name. I can tell more if your provide me a link on your deployment.
